Various/ausbaxter_Render item columns.lua

```diff
@@ -0,0 +1,3878 @@
+--@description ausbaxter_Render item columns
+--@version 1.0
+--@author ausbaxter
+--@about
+--    # Render Item Columns
+--
+--    This package provides a gui interface to render columns of selected items. Has optional tail length, rename capabilities, and its also possible to add regions based on item columns.
+--    GUI created using Lokasenna's GUI 2.0. https://forum.cockos.com/showthread.php?t=177772
+--@provides
+--    ausbaxter_Render item columns.lua
```

Comment:
This provides tag is not necessary in this case because the current script file is implicitly added to the package's file list. However it doesn't hurt to explicitly list it.